src/server/manager.rs

```diff
@@ -32,6 +32,14 @@ pub fn run(server_address: &str) {
     let _ = System::new("http-server");
     HttpServer::new(|| {
         App::new()
+            // Load static files from configured www-path
+            .service(
+                actix_files::Files::new(
+                    "webrtc",
+                    format!("{path}/webrtc/", path = cli::manager::www_path()),
+                )
+                .index_file("index.html"),
+            )
```

Comment:
Note that this is being done before the paperclip wrapper because it doesn't seem to support such a thing. (see [here](https://github.com/paperclip-rs/paperclip/issues/365)).

To have the webrtc listed on the docs.json, I added a route (which is never called because of some kind of priority on this chain, I guess) also after the paperclip wrapper.

Reply:
We are ok to no having it on the swagger api, since this is a webpage and not a REST API we should be fine.

Reply:
Oh, okay, that makes sense, so I just got rid of that workaround.